src/styles/partials/template/_sidebar.scss

```diff
@@ -20,6 +20,15 @@
     font-size: $smallest-heading-font-size;
   }
 
+  .dg_sidebar__content {
+    padding: $padding-small $padding-large;
+  }
+
+  .dg_sidebar__content_gray {
```

Comment:
how does this differ from the rule above this? I think maybe we could name this a little better to describe it.

Reply:
Essentially all it does is add the option for a gray background. Maybe .dg_sidebar___content_gray_background? A bit long though.